src/app/plan/duck/reducers.ts

```diff
@@ -396,7 +381,6 @@ const planReducer = (state = INITIAL_STATE, action) => {
     case PlanActionTypes.SET_CURRENT_PLAN: return setCurrentPlan(state, action);
     case PlanActionTypes.SET_LOCKED_PLAN: return setLockedPlan(state, action);
     case PlanActionTypes.PLAN_CLOSE_AND_DELETE_SUCCESS:
-      return planCloseAndDeleteSuccess(state, action);
```

Comment:
Is this correct?

Reply:
![bitmoji](https://render.bitstrips.com/v2/cpanel/05ce7e0e-688b-4fe4-a3e2-51692fb6c7e2-0c62b4b5-eb2f-46ae-82aa-ce8e6130af1b-v1.png?transparent=1&palette=1&width=246)